schema.rst

```diff
@@ -205,6 +205,15 @@ Attribute names are case sensitive.
   Specifies the minimum C++ standard/runtime library version
   required by the package's CABI components.
 
+.. ----------------------------------------------------------------------------
+.. cps:attribute:: cpp_module_metadata
+  :type: string
+  :context: component
+
+  Specifies the path to a C++ module metadata file (also known as a
+  "P3286" file) necessary for consuming C++ interface units for
+  pre-built libraries.
```

Comment:
Per the contributing guidelines, please "try to break at 'natural' boundaries". In particular, before the parenthetical rather than in the middle thereof is preferred.

```suggestion
  Specifies the path to a C++ module metadata file
  (also known as a "P3286_" file) 
  necessary for consuming C++ interface units for pre-built libraries.
```

Also, please make `P3286` a link to https://wg21.link/p3286. (Please add the link definition to the bottom with the other link definitions.)

I'm not sure this text is adequately descriptive, but I don't have anything better offhand, and in practice I expect users to understand what this means. Therefore, I'm not going to worry about it now; we can always improve the prose later as a non-functional change.